gradle/lucene-dev/lucene-dev-repo-composite.gradle

```diff
@@ -130,6 +130,21 @@ if (luceneDevRepo != null) {
   } else {
     // We're being applied at build-time and Lucene development repository exists. Configure
     // certain aspects of the build so that things work with it.
+    
+    // replace luceneBaseVersionProvider by one evaluating the included build:
+    configure(rootProject) {
+      def line = new File(gradle.includedBuild('lucene').projectDir, 'build.gradle').readLines("UTF-8").find { it =~ /\bbaseVersion\s*=\s*['"]/ }
```

Comment:
Yep. included builds don't really have a notion of "version" - they only have coordinates (they're treated much like subprojects). I don't know if there is a better solution than that presented here.